### PR TITLE
6 footer

### LINKED ISSUE #6: Footer
Create static footer with legal links + infomation like:

https://local.ubs.com/westchester-purchase-ny/Index.htm

### PR DESCRIPTION
**Pull request by kesiah**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--edge-delivery-ubs-faweb-demo--netcentric.aem.live/
- After: https://6-footer--edge-delivery-ubs-faweb-demo--netcentric.aem.live/

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -4,18 +4,59 @@ footer {
...
-  margin: 0;
-}
+  color: var(--footer-text-color);
+
+  > div {
+    margin: auto;
+    max-width: var(--content-max-width);
+    padding: 40px 24px 24px;
+  
+    @media (width >= 900px) {
```
Comment (dragoss-nc) [inline blocks/footer/footer.css]: If we go for the prefix notation, we will have to change this, or allow both.

File: styles/styles.css [hunk 2]
```diff
@@ -106,13 +106,7 @@
   --col-neutral90: #1C1C1C;
 
   /* Spacing */
-  --grid-max-width: 1290px;
```
Comment (dragoss-nc) [inline styles/styles.css]: I think we will need variables for content side paddings as well.

**Review by dragoss-nc — APPROVED** [review]

File: styles/styles.css (hunk 2 above)
Comment (dragoss-nc) [reply styles/styles.css]: Also the value should actually be 1290px, not 1200px. We should update everywhere.

File: blocks/footer/footer.css [hunk 3]
```diff
@@ -4,18 +4,59 @@ footer {
...
 
-footer .footer p {
-  margin: 0;
-}
+  color: var(--footer-text-color);
+
+  > div {
+    margin: auto;
+    max-width: var(--content-max-width);
+    padding: 40px 24px 24px;
```
Comment (kesiah) [inline blocks/footer/footer.css]: check variables after https://github.com/Netcentric/edge-delivery-ubs-faweb-demo/pull/17 is merged

**Pull request merged**